runtime/src/iree/builtins/device/BUILD.bazel

```diff
@@ -51,6 +51,13 @@ endif()
 
 # TODO(benvanik): rule for building a matrix of bitcode files.
 
+iree_bitcode_library(
+    name = "libdevice_device_aarch64_sme",
```

Comment:
`libdevice_aarch64_sme` for consistency?

Reply:
Whoops, that was a typo! I was blind to the double 'device' there :sweat_smile: I'll fix that up :+1: